packages/liburcu/liburcu.spec

```diff
@@ -0,0 +1,67 @@
+Name: %{_cross_os}liburcu
+Version: 0.14.0
+Release: 1%{?dist}
+Summary: Library for userspace RCU
+License: LGPL-2.1-or-later
+URL: http://liburcu.org
+Source0: http://lttng.org/files/urcu/userspace-rcu-latest-0.14.tar.bz2
```

Comment:
nit:
```suggestion
Version: 0.14
Release: 1%{?dist}
Summary: Library for userspace RCU
License: LGPL-2.1-or-later
URL: http://liburcu.org
Source0: http://lttng.org/files/urcu/userspace-rcu-latest-%{version}.tar.bz2
```

Reply:
The version is nonstandard, the version *is* 0.14.0 but the tarball that is downloaded is named 0.14. So to use `%{version}` during the the `autosetup` step, it needs to have the .0 in it. So either we have to do conversion in the `Source0` or the `autosetup` and I felt like it is "more correct" to change the name from the Source than in the `autosetup`. I can switch it around though if you feel its better this way.

Reply:
The way you currently have it is fine. I wasn't aware of how the version is used elsewhere.